master/getting-started/docker/installation/manual.md

```diff
@@ -8,16 +8,16 @@ Calico runs as a Docker container on each host. The `calicoctl` command line too
 
 1. Download the calicoctl binary:
 
-	 ```
-   sudo wget -O /usr/local/bin/calicoctl http://www.projectcalico.org/builds/calicoctl
```

Comment:
For proof, view the github rendering of these changes. It shows the 2nd item as `1` since it thinks the subcontent is new content.

(It's really difficult to explain what I mean, hope I'm making sense)